src/legacy/server/saved_objects/saved_objects_mixin.js

```diff
@@ -141,6 +144,12 @@ export function savedObjectsMixin(kbnServer, server) {
     setScopedSavedObjectsClientFactory: (...args) => provider.setClientFactory(...args),
     addScopedSavedObjectsClientWrapperFactory: (...args) =>
       provider.addClientWrapperFactory(...args),
+    importExport: {
+      importSavedObjects,
+      getSortedObjectsForExport,
+      objectsToNdJson,
```

Comment:
The contract between import and export is via a `ndjson` string within a `Readable` stream. This PR does not attempt to change that contract, but as a result, we end up having to expose this converter to the outside world. For this POC, I'm going for a low-touch implementation. We can revisit if necessary